etc/my.cnf

```diff
@@ -27,6 +27,6 @@ user=mysql
 # Disabling symbolic-links is recommended to prevent assorted security risks
 symbolic-links=0
 
-log-error=/var/log/mysqld.log
+#log-error=/var/log/mysqld.log
```

Comment:
intended?

Reply:
Actually in the   "latest" Tag image  
when we bring up the container  it complains with the error:
'/var/lib/mysqld.log'   permission denied,    that was in the latest tag  
seems like its working in  8.0
i would change this

Reply:
changes added